File: go/wam/4784.go
const WamWebDbVersionsSource = 4784 // channel: regular

type WebDbVersionsSourceEvent struct {
	WebDbVersionSource WEBDBVERSIONSOURCETYPE `wam:"web_db_version_source"` // field 1
	WebSchemaInitiator WEBSCHEMAINITIATORTYPE `wam:"web_schema_initiator"` // field 2
}

File: ts/wam/4784.ts
export const WamWebDbVersionsSource = 4784 as const

export interface WebDbVersionsSourceEvent {
  /** field 1, wire `web_db_version_source` */
  webDbVersionSource?: WEBDBVERSIONSOURCETYPE
  /** field 2, wire `web_schema_initiator` */
  webSchemaInitiator?: WEBSCHEMAINITIATORTYPE
}